target/linux/ramips/image/mt7621.mk

```diff
@@ -443,66 +443,65 @@ define Device/netgear_ex6150
 endef
 TARGET_DEVICES += netgear_ex6150
 
-define Device/netgear_r6220
+define Device/netgear_r6xxx
```

Comment:
I would go with netgear_sercomm_nand here.

The `<digit>xxx` scheme is becoming more and more confusing in many targets.

Reply:
Fine with me.